app/models/author.rb

```diff
@@ -0,0 +1,4 @@
+class Author < ApplicationRecord
+  has_many :book_authors, dependent: :destroy
+  has_many :books, dependent: :destroy
```

Comment:
```suggestion
  has_many :books, through: :book_authors
```
ri mới đúng chứ em

author và book quan hệ nhiều nhiều thông qua book_author